README.md

```diff
@@ -331,6 +348,9 @@ console.log(stringify(obj)) // '{"id":18446744073709551615}'
 If you want to squeeze a little bit more performance out of the serialisation, at the cost of readability in the generated code, you can pass `uglify: true` as an option.
 Note that you have to manually install `uglify-es` in order for it to work. Only version 3 is supported.
 Example:
+
+Note that if you are using Node 8.3.0 or newer, performance gains from using uglify are negligible.
+
```

Comment:
So instead of "negligible" just say "no performance gains" and link to that blogpost?